apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations/[evaluationId]/_components/EvaluationResults/EvaluationResultInfo/index.tsx

```diff
@@ -21,12 +22,15 @@ function useFetchDocumentLog({ documentLogId }: { documentLogId: number }) {
     data: documentLog,
     isLoading,
     error,
-  } = useSWR<DocumentLogWithMetadata>(
+  } = useSWR<DocumentLogWithMetadataAndError>(
     ['documentLogs', documentLogId],
     useCallback(async () => {
```

Comment:
hmm can we use useFetcher? not sure why it ain't used here, probably missed it somehow in my refactor

Reply:
I'll do this in a smaller PR when we're sure these changes are working